paket.dependencies

```diff
@@ -65,7 +65,7 @@ nuget Serilog.Settings.Configuration 3.3.0
 nuget Serilog.Sinks.Seq 5.1.1
 nuget Swashbuckle.AspNetCore.SwaggerUI 6.2.3
 
-nuget FeatureToggle.Net6 1.0.1
+nuget FeatureToggle.Common 4.0.2
```

Comment:
why change from .Net6?

Reply:
When using the `IFeatureToggle` interface in the Backoffice API there became a conflict of 2 assemblies containing it. The FeatureToggle.Common.Net6 (v1.0.0.0) and FeatureToggle.Common (v4.0.2.0).
I then checked streetname-registry and saw that their `IFeatureToggle` came from the FeatureToggle.Common (v4.0.2.0) assembly.

Now upon further investigation I see that actually the FeatureToggle.Common assembly comes from the nuget `Be.Vlaanderen.Basisregisters.DataDog.Tracing.Autofac` even though it doesn't have the FeatureToggle.Common listed as an dependency.
`.nuget\packages\be.vlaanderen.basisregisters.datadog.tracing.autofac\5.0.0\lib\net6.0\FeatureToggle.Common.dll`

Reply:
so why not use .Net6 ?